plugins/webp-uploads/hooks.php

```diff
@@ -519,65 +519,24 @@ function webp_uploads_remove_sources_files( int $attachment_id ): void {
  * would rely on `wp_filter_content_tags()`.
  *
  * @since 1.0.0
+ * @since n.e.x.t Added the `$context` and `$attachment_id` parameters.
  *
  * @see wp_filter_content_tags()
  *
- * @param string|mixed $content The content of the current post.
+ * @param string $filtered_image Full img tag with attributes that will replace the source img tag.
+ * @param string $context        Additional context, like the current filter name or the function name from where this was called.
+ * @param int    $attachment_id  The image attachment ID. May be 0 in case the image is not an attachment.
  * @return string The content with the updated references to the images.
  */
-function webp_uploads_update_image_references( $content ): string {
-	if ( ! is_string( $content ) ) {
-		$content = '';
-	}
-
+function webp_uploads_update_image_references( string $filtered_image, string $context, int $attachment_id ): string {
```

Comment:
Since the signature is already not compatible with the old function, we can also come up with a better name given the different usage. For example `webp_uploads_filter_image_tag()`.